app/models/job.rb

```diff
@@ -1,6 +1,11 @@
 class Job < ActiveRecord::Base
   extend Enumerize
-  STATUSES = [:draft, :published, :archived]
+  STATUSES = %w(draft published archived)
   enumerize :state, in: STATUSES
```

Comment:
Smell: we're using 2 words ('status' and 'state') to refer to the same concept.

Maybe we could simply say this?

```ruby
STATES = %w(draft published archived)
enumerize :state, in: STATES
```